pkg/binding/registry/registry.go

```diff
@@ -31,15 +31,19 @@ func New() Registry {
 			},
 			schema.GroupVersionKind{Group: "pxc.percona.com", Version: "v1-8-0", Kind: "PerconaXtraDBCluster"}: {
 				"service.binding/type":     "mysql",
+				"service.binding/database": "mysql",
```

Comment:
This is the name of the database, right?
Is that going to be always `mysql`?

Reply:
For this particular `GroupVersionKind`, I think it's always going to be a mysql db that gets created.  I see no reason for Percona to retroactively change this to be something else.